resources/js/beatmap-discussions/nominations.tsx

```diff
@@ -0,0 +1,680 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+import BeatmapsOwnerEditor from 'beatmap-discussions/beatmaps-owner-editor';
+import LoveBeatmapDialog from 'beatmap-discussions/love-beatmap-dialog';
+import { Nominator } from 'beatmap-discussions/nominator';
+import PlainTextPreview from 'beatmap-discussions/plain-text-preview';
+import BigButton from 'components/big-button';
+import DiscreteBar from 'components/discrete-bar';
+import Modal from 'components/modal';
+import StringWithComponent from 'components/string-with-component';
+import TimeWithTooltip from 'components/time-with-tooltip';
+import UserLink from 'components/user-link';
+import { BeatmapsetDiscussionJsonForShow } from 'interfaces/beatmapset-discussion-json';
+import BeatmapsetEventJson from 'interfaces/beatmapset-event-json';
+import { BeatmapsetNominationsInterface, BeatmapsetStatus } from 'interfaces/beatmapset-json';
+import BeatmapsetWithDiscussionsJson from 'interfaces/beatmapset-with-discussions-json';
+import GameMode from 'interfaces/game-mode';
+import UserJson from 'interfaces/user-json';
+import { route } from 'laroute';
+import { findLast } from 'lodash';
+import { action, makeObservable, observable } from 'mobx';
+import { observer } from 'mobx-react';
+import { deletedUser } from 'models/user';
+import moment from 'moment';
+import core from 'osu-core-singleton';
+import * as React from 'react';
+import { onError } from 'utils/ajax';
+import { canModeratePosts, makeUrl } from 'utils/beatmapset-discussion-helper';
+import { nominationsCount } from 'utils/beatmapset-helper';
+import { classWithModifiers } from 'utils/css';
+import { joinComponents, trans, transExists } from 'utils/lang';
+import { pageChange } from 'utils/page-change';
+import { presence } from 'utils/string';
+import { wikiUrl } from 'utils/url';
+import CurrentDiscussions from './current-discussions';
+
+const bn = 'beatmap-discussion-nomination';
+const flashClass = 'js-flash-border--on';
+export const hypeExplanationClass = 'js-hype--explanation';
+const nominatorsVisibleBeatmapStatuses: Readonly<BeatmapsetStatus[]> = Object.freeze(['wip', 'pending', 'ranked', 'qualified']);
```

Comment:
Using set is a bit more in line with how it's used maybe?
```suggestion
const nominatorsVisibleBeatmapStatuses = new Set<BeatmapsetStatus>(['wip', 'pending', 'ranked', 'qualified']);
```